packages/plugin-react-swc/package.json

```diff
@@ -1,15 +1,36 @@
 {
   "name": "@vitejs/plugin-react-swc",
   "version": "3.8.1",
+  "license": "MIT",
+  "author": "Arnaud Barré (https://github.com/ArnaudBarre)",
+  "description": "Speed up your Vite dev server with SWC",
+  "keywords": [
+    "vite",
+    "vite-plugin",
+    "react",
+    "swc",
+    "react-refresh",
+    "fast refresh"
+  ],
   "type": "module",
-  "private": true,
   "scripts": {
     "dev": "tsx scripts/bundle.ts --dev",
     "build": "tsx scripts/bundle.ts",
     "test": "playwright test",
     "release": "pnpm build && tsx scripts/release.ts"
   },
-  "packageManager": "pnpm@10.6.4",
+  "engines": {
+    "node": "^14.18.0 || >=16.0.0"
```

Comment:
I'm not sure I wish to align this. I think node 14 was already deprecated when the plugin started.

Reply:
I guessed the plugin supports it from this line.
https://github.com/vitejs/vite-plugin-react-swc/blob/89710412d579cb0e305b0495dd1985fb056b2a74/scripts/bundle.ts#L14
I'll remove it 👍

Reply:
Ah true there was not a major since 2022 so maybe it's actually supported.